internal/kldcontracts/remoteregistry.go

```diff
@@ -148,7 +148,7 @@ func (rr *remoteRegistry) loadFactoryFromURL(baseURL, ns, lookupStr string) (*de
 	err = json.Unmarshal([]byte(abiString), &abi)
 	if err != nil {
 		log.Errorf("GET %s <-- !Failed to decode ABI: %s\n%s", queryURL, err, abiString)
-		return nil, fmt.Errorf(genericRegistryResponseErrorMsg)
+		return nil, klderrors.Errorf(klderrors.RemoteRegistryLookupDecodeABIFailed)
```

Comment:
```suggestion
		return nil, klderrors.Errorf(klderrors.RemoteRegistryLookupDecodeABIFailed, err)
```

Reply:
So far, we've deliberately not returned the response over the REST API on this one. So the 'generic' error is intentionally retained

Reply:
going to rename to avoid confusion though